plugin-tester-java/src/main/java/example/myapp/helloworld/GreeterClient.java

```diff
@@ -26,14 +26,10 @@ public static void main(String[] args) throws Exception {
     String serverHost = "127.0.0.1";
     int serverPort = 8080;
 
-    GrpcClientSettings settings = GrpcClientSettings.create(serverHost, serverPort)
-      // Note: In this sample we are using a dummy TLS cert so we need to fake the authority
-      .withOverrideAuthority("foo.test.google.fr")
-      .withTrustedCaCertificate("ca.pem");
-
     ActorSystem system = ActorSystem.create("HelloWorldClient");
     Materializer materializer = ActorMaterializer.create(system);
 
+    GrpcClientSettings settings = GrpcClientSettings.create("helloworld.GreeterService", system);
```

Comment:
Can we replace `"helloworld.GreeterService"` with `GreeterService.name` or is the string used on purpose to ensure the correct value is used?

Reply:
That's how we will do it in the autogenerated client providers in play, but maybe explicitly stating it is more clear in an example like this.